src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,321 @@
+package duke;
+
+import java.util.ArrayList;
+import duke.exceptions.InvalidCommandException;
+import duke.exceptions.InvalidFileException;
+import duke.exceptions.InvalidTaskIndexException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+
+class Duke {
+    private static final int LINE_CHAR_NUM = 40;
```

Comment:
I think adding a Javadoc comment to explain `40` would be very helpful as `LINE_CHAR_NUM` is not very informative.